openshift-maven-plugin/it/src/it/simple-with-route-flag-false/invoker.properties

```diff
@@ -13,5 +13,5 @@
 #
 
 invoker.goals.1=clean oc:resource
-invoker.mavenOpts=-Djkube.verbose -Djkube.offline=true -Dfabric8.openshift.generateRoute=false
+invoker.mavenOpts=-Djkube.verbose -Djkube.offline=true -Djkube.openshift.generateRoute=false
```

Comment:
I'm really curious why this wasn't causing a test failure

Reply:
Our current integration test suite doesn't seem to be verifying missing elements from expected yaml. It only verifies elements present. I had added this assertion to make sure that test fails:

https://github.com/eclipse/jkube/blob/ded7049e0ff253e04ebf1fddfc265bf25f1b636b/openshift-maven-plugin/it/src/it/simple-with-route-flag-false/verify.groovy#L23